server/services/v1/mutator_test.go

```diff
@@ -16,14 +16,191 @@ package v1
 
 import (
 	"bytes"
+	"fmt"
 	"testing"
+	"time"
 
 	jsoniter "github.com/json-iterator/go"
 	"github.com/stretchr/testify/require"
 	"github.com/tigrisdata/tigris/lib/json"
 	"github.com/tigrisdata/tigris/schema"
 )
 
+func TestMutateSetDefaults(t *testing.T) {
+	reqSchema := []byte(`{
+		"title": "t1",
+		"properties": {
+			"id": {
+				"type": "integer"
+			},
+			"double_f": {
+				"type": "number",
+				"default": 1.5
```

Comment:
I think it is important to add a test here to make sure that a field with `now()` and the createdAt are the same value. Users are really surprised if the timestamps used in a mutation operation are not all the same if they are set in that operation.

Reply:
Yes, it will be added.

Reply:
Done, added an integration test(documents_test.go) that is validating that now() and createdAt are the same values.